docs/guide/deployment/resource-aggregate.md

```diff
@@ -154,6 +154,7 @@ Plgd cloud uses NATS messaging system as a event bus.
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
 | `clients.eventBus.nats.url` | string | `URL to nats messaging system.` | `"nats://localhost:4222"` |
+| `clients.eventBus.nats.jetstream `| bool | `If true, events will be published to jetstream.` | `false` |
```

Comment:
Does the connection string stay the same (just pointing to jetstream? is there any difference?)? It's still eventbus.nats which needs to be configured?

Reply:
No. It same for jetstream.